### DISCUSSION TITLE
Base config from repo not working and sending errors

### DISCUSSION THREAD

**Discussion by MartinFillon** [helix-editor/helix#9609] [General]
### Summary

File is the raw one from main
I have errors such as : 
```
Bad language config: unknown field `persistent-diagnostic-sources`, expected one of `name`, `language-id`, `scope`, `file-types`, `shebangs`, `roots`, `comment-token`, `text-width`, `soft-wrap`, `auto-format`, `formatter`, `diagnostic-severity`, `grammar`, `injection-regex`, `language-servers`, `indent`, `debugger`, `auto-pairs`, `rulers`, `workspace-lsp-roots`
in `language`
```
Or:
```
Bad language config: unknown key in `file-types` list: glob
in `language.file-types`
```

### Reproduction Steps

 

I tried this:

1. copy file from repo
2. `hx`

I expected this to happen:
no error displayed
Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

Log is not useful as not related :

```
2024-02-12T12:55:07.711 helix_view::editor [ERROR] Failed to initialize the language servers for `source.toml` - `taplo` { cannot find binary path }
```

</details>

### Platform

Linux

### Terminal Emulator

kitty

### Installation Method

yay

### Helix Version

helix 23.10

**Comment by TornaxO7** [comment]
This error:
```
Bad language config: unknown key in `file-types` list: glob
in `language.file-types
```
should come from [this PR](https://github.com/helix-editor/helix/pull/8006) and I assume the other error comes from another recent PR as well. So you can either use `helix` from the `master` branch or remove the invalid lines from your config.

**Comment by MartinFillon** [comment]
Yeah I guess I am gonna try helix from master.

**Comment by pascalkuthe** [comment]
You should never copy the languages.to.l from the repo it's built into the binary

**Comment by sgon00** [comment]
When will we have a release which supports `glob`? There is no release for a long time. Thanks a lot.
